File: Algorithm/programmers/Level1/나누어 떨어지는 숫자 배열.py
def solution(arr, divisor):
    answer = []
    for num in arr:
        if num % divisor == 0:
            answer.append(num)

    return sorted(answer) if len(answer) > 0 else [-1]


def solution(arr, divisor):
    return sorted([n for n in arr if n % divisor == 0]) or [-1]

File: Algorithm/programmers/Level1/나누어 떨어지는 숫자 배열.js
function solution(arr, divisor) {
  var answer = [];
  // 오름차순 정렬
  arr.sort((a, b) => a - b);
  arr.forEach((v) => {
    if (v % divisor === 0) answer.push(v);
  });
  return answer.length > 0 ? answer : [-1];
}

function solution(arr, divisor) {
  let answer = arr.filter((v) => v % divisor === 0);
  return answer.length > 0 ? answer.sort((a, b) => a - b) : [-1];
}
